### PR TITLE
docs(others): Added few pages and modified some content in doc

### PR DESCRIPTION
**Pull request by sussii**
Added 3 pages:
- Appendix-built-in data identifier
- Appendix-Permissions
- About premium edition

Modified few pages:
- Data identifiers, added detail page and an example
- Data catalog sync page
- Overview image
- Data catalog create page

### CODE REVIEW

**Review by yanbasic — APPROVED** [review]
LGTM

File: docs/en/user-guide/data-catalog-sync.md [hunk 1]
```diff
@@ -1,29 +1,32 @@
...
-
 ## When are the data catalogs synchronized with data source?
 
 The solution synchronizes the data catalogs with data source in the following situations:
+- Sync to data catalog (manual). Please refer to [Connect to data source](data-catalog-create.md)
+- Run sensitive data discovery job (automatic)
 
-- when data catalog is created
-- when sensitive data discovery job runs
+!!! Info "For more information"
```
Comment (rrxie) [inline docs/en/user-guide/data-catalog-sync.md:10]: !!! Important "Important"

File: docs/en/user-guide/data-identifiers.md [hunk 2]
```diff
@@ -1,16 +1,66 @@
-A data identifier is a rule to detect sensitive data. The solution allows you to manage all the identifiers. You can view the built-in identifiers or create custom identifiers with RegEx or keywords.
+Data identifiers are rules for detecting sensitive data. First, you need to understand what type of data is defined as sensitive in your company, and the identification rules for this sensitive data. Only sensitive data that can be defined using regular expressions or AI can be identified through technical means (such as through this SDP solution).
```
Comment (rrxie) [inline docs/en/user-guide/data-identifiers.md:1]: 这里的 “AI” 代表的是？

File: docs/zh/user-guide/appendix-permissions.md [hunk 3]
```diff
@@ -1,16 +1,16 @@
-# 附录：Agent CloudFormation 堆栈的权限
+# 附录：代理 CloudFormation 堆栈的权限
 
-解决方案遵循最小权限原则，仅创建所需权限。权限的高级概念：
+本解决方案在部署代理 CloudFormation 模板时，遵循最小权限原则为受监控帐户授予权限。代理 CloudFormation 堆栈基本上会设置 AWS 角色和策略，以授权 SDPS 在目标帐户中获取必要的权限。
```
Comment (rrxie) [inline docs/zh/user-guide/appendix-permissions.md:3]: SDP，或者 解决方案

**Review by rrxie — APPROVED** [review]

**Pull request merged**